lang/en/settings.php

```diff
@@ -6,6 +6,7 @@
  * @author Wolfgang Gassler <wolfgang@gassler.org>
  */
 
+$lang['pingPushHost'] = 'Ping the push host, to make sure it is available before pushing. empty to disable';
```

Comment:
Please change wording to: `Ping the push host, to make sure it is available before pushing. Disabled if empty.`